### DISCUSSION TITLE
Crowd simulation with dynamically rebuilt navigation meshes and Motion Matching

### DISCUSSION THREAD

**Discussion by GeorgeS2019** [GuilhermeGSousa/godot-motion-matching#49] [Ideas]
@smix8

I need your feedback what are the possible demo combining crowd simulation where individuals are **self-aware** of other individuals in the crowd

WIP demo
https://github.com/GuilhermeGSousa/godot-motion-matching/discussions/34

### References
@bgaechter

https://forum.godotengine.org/t/dynamically-update-navigationregion3d-navigationmesh/63865
https://www.reddit.com/r/godot/comments/e7g3rx/crowd_simulation_with_dynamically_rebuilt/

@popitchus
https://www.reddit.com/r/godot/comments/1ai0b1x/seeking_advice_implementing_custom_crowd/

https://github.com/sky94520/godot-rvo2/issues/2

https://github.com/WarpWorld/crowd-control-unity-plugin
https://github.com/plaans/gobot-sim/issues/64

**Comment by GeorgeS2019** [comment]
## A REVIEW OF SOFTWARE FOR CROWD SIMULATION
extension://bfdogplmndidlpjfhoijckpakkdjkkil/pdf/viewer.html?file=https%3A%2F%2Furban-analytics.github.io%2Fdust%2Fdocs%2Fped_sim_review.pdf

**Comment by fire** [comment]
Crowd simulation is combining navigation and large crowd rendering. It isn't directly motion matching. So I would prefer if this was a discussion.

**Reply by GeorgeS2019** [reply]
If you check Unreal and Unity Motion matching, this is an active area of development
